vortex-dtype/src/ptype.rs

```diff
@@ -104,7 +104,9 @@ macro_rules! match_each_integer_ptype {
             PType::U16 => __with__! { u16 },
             PType::U32 => __with__! { u32 },
             PType::U64 => __with__! { u64 },
-            _ => panic!("Unsupported ptype {}", $self),
+            PType::F16 =>  panic!("Unsupported ptype f16"),
+            PType::F32 =>  panic!("Unsupported ptype f32"),
+            PType::F64 =>  panic!("Unsupported ptype f64"),
```

Comment:
was this necessary?

Reply:
to remove any string concat at compile time

Reply:
(So max_value can be a const fn since format! isn't const)